.husky/pre-commit

```diff
@@ -1,4 +1 @@
-#!/bin/sh
```

Comment:
These husky changes areas outlined in the release notes

https://github.com/typicode/husky/releases/tag/v9.0.1